src/OpenTK.Mathematics/Vector/Vector4h.cs

```diff
@@ -120,49 +120,55 @@ public Vector4h(float x, float y, float z, float w)
         /// <summary>
         /// Initializes a new instance of the <see cref="Vector4h"/> struct.
         /// </summary>
-        /// <param name="v">The <see cref="Vector4"/> to convert.</param>
-        public Vector4h(Vector4 v)
+        /// <param name="xy">The x and y components of the Vector4h.</param>
+        /// <param name="z">The z component of the Vector4h.</param>
+        /// <param name="w">The w component of the Vector4h.</param>
+        public Vector4h(Vector2h xy, float z = default, float w = default)
```

Comment:
We want another version of this ctor that takes `Vector2, float, float`.

Reply:
See https://github.com/opentk/opentk/pull/1419#discussion_r845903852